fish_prompt.fish

```diff
@@ -895,16 +896,30 @@ function __bobthefish_prompt_desk -S -d 'Display current desk environment'
     set_color normal
 end
 
-function __bobthefish_prompt_nvm -S -d 'Display current node version through NVM'
-    [ "$theme_display_nvm" = 'yes' -a -n "$NVM_DIR" ]
+function __bobthefish_prompt_node -S -d 'Display current node version'
+    [ "$theme_display_node" = "yes" -o "$theme_display_nvm" = 'yes' ]
```

Comment:
This supports both `theme_display_node` and `theme_display_nvm` for backwards compatibility.